### ISSUE TITLE
Handling multiple storage buckets - uploading a file

### ISSUE THREAD

**Issue by K0ff33** [invertase/react-native-firebase#776]
### Issue

I have a problem with uploading a file to a different bucket within the same account. 

I've tried following the [official docs](https://firebase.google.com/docs/storage/web/start#use_multiple_storage_buckets) with the following code:
```javascript
const storage = firebase.storage("gs://my-custom-bucket");

```
but receive the error: `Invalid FirebaseApp instance passed to firebase.storage(app <--)`

Also when trying to use:
```javascript
firebase.storage().refFromURL("gs://my-custom-bucket/path-to.file")
```
the app crashes on iOS, while Android throws an error: `The supplied bucketname does not match the storage bucket of the current instance`.

Uploading files to the default bucket works well.

### Environment

1. Application Target Platform: Both

2. Development Operating System: macOS High Sierra

3. Build Tools: not relevant

4. React Native version: 0.51.0

5. RNFirebase Version: 3.2.2

6. Firebase Module: storage

**Comment by K0ff33** [comment]
Sorry, duplicate of #775

**Comment by VinodhPIT** [comment]
I'm also facing  the same issue  any solutions

**Comment by mikehardy** [comment]
@VinodhPIT you have posted a comment with no details on an issue that was closed 4.5 years ago. That is very confusing behavior

**Comment by VinodhPIT** [comment]
Now i'm  working  on  an  old react native project  ,they are using  firebase V5 and now i'm  not able to call multiple buckets, only default bucket is working.I need to store some particular  images to different bucket . 

Error: Invalid FirebaseApp instance passed to firebase.storage(app <--)

Any solution for this ?
